src/FBX2glTF.cpp

```diff
@@ -310,7 +323,7 @@ int main(int argc, char* argv[]) {
   if (!texturesTransforms.empty()) {
     raw.TransformTextures(texturesTransforms);
   }
-  raw.Condense();
+  raw.Condense(gltfOptions.maxSkinningWeights, gltfOptions.normalizeSkinningWeights);
```

Comment:
Random thought: I'm starting to think we should have `RawOptions`, `FbxOptions` and `GltfOptions` interfaces, each one defined in the relevant subdirectory, then a concrete class that implements all three, and finally instantiate e.g. `RawModel` with an instance of the option class. It could be argued that it's more elegant to send to each method exactly the options it needs, but there's an equally valid point that once a method starts accepting more than 4 parameters, it's lost most of its ability to clean things up rather than muddy them.

(This method is obviously fine, just thinking outloud.)

Reply:
Yes, this was actually something I was considering as well, and influenced some of the design choices in the implementation. We could have gone a different route and sent options all the way to the gltf-agnostic Fbx2Raw stuff and had the Vec4 structures set up right there. But then I think it's good to keep that part of the code agnostic to gltf limitations and structures anyway.